src/main/resources/application-test.properties

```diff
@@ -3,6 +3,11 @@ quarkus.smallrye-openapi.info-title=Cryostat API (test)
 cryostat.discovery.jdp.enabled=true
 cryostat.discovery.podman.enabled=true
 cryostat.discovery.docker.enabled=true
+cryostat.discovery.kubernetes.enabled=true
```

Comment:
I think you can remove these lines from the `-test` and `-dev` profiles - these are applied as overrides on top of the default `application.properties`, so if the desired values are the same then they can be left off so they default to the standard profile.